BashDynDnsChecker.sh

```diff
@@ -845,31 +845,33 @@ if [ "$current_ip" != "$old_ip" ]; then
                 exit 11
                 ;;
     esac
-    case $first_part in
-        10)
-                msg_error "IP address $current_ip is part of private network"
-                exit 11
-                ;;
-        169)
-            if [ "$second_part" == "254" ]; then
-                msg_error "IP address $current_ip is part of private network"
-                exit 11
-            fi
-                ;;
-        172)
-            if [ $second_part -ge 16 ] && [ $second_part -le 31 ]
-             then
-                msg_error "IP address $current_ip is part of private network"
-                exit 11
-            fi
-                ;;
-        192)
-            if [ "$second_part" == "168" ]; then
-                msg_error "IP address $current_ip is part of private network"
-                exit 11
-            fi
-                ;;
-    esac
+    if [ "$BDDC_ALLOW_PRIVATE" -ne 1 ]; then
```

Comment:
ok. this might be useful for local dns requests. 
not sure if this should be in there. If someone REALLY wants their local IP on a DNS server they might just kill this IP validation section.